Polyhedron/demo/Polyhedron/Viewer.cpp

```diff
@@ -30,6 +30,8 @@ class Viewer_impl {
   mutable std::vector<QOpenGLShaderProgram*> shader_programs;
   QMatrix4x4 projectionMatrix;
   void setFrustum(double l, double r, double t, double b, double n, double f);
+  QImage* takeSnapshot(Viewer* viewer, int quality, int background_color, QSize size, double oversampling, bool expand);
+  void sendSnapshotToClibboard(Viewer*);
```

Comment:
@maxGimeno: I agree with Sébastien. It is spelled "clipboard" with a 'p'. A clipboard is [a board with a clip](https://www.google.fr/search?q=clipboard): ![clipboard image].

[clipboard image]: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c0/Wood-clipboard.jpg/480px-Wood-clipboard.jpg